### ISSUE TITLE
Experiment mode: Every Word is a button

### ISSUE THREAD

**Issue by johnwdubois** [johnwdubois/rezonator#654]
**Background**
It is useful to use Rezonator to do experiments (whether on MTurk or not). For example, one kind of experiment would be to ask users to listen to the audio for each Turn, and then divide the Turn into Intonation Units. Thus, instead of showing them the Intonation Units from the beginning, this would require them to listen and then decide which word is the last word of each Intonation Unit.

**What to do**
Create an "Experiment mode". Once the user selects this mode:
1. In the Rezonator Main screen, display each Turn as a long paragraph of words (with word-wrap). (See #653, #482 )
2. Every Word acts as a button, which will perform an action as specified for a given Experiment.
3. In this case, clicking on a Word makes that Word the last Word of the Unit (i.e. it sets the value of `LastWordOfUnit` = TRUE)
4. This Unit is now displayed on a single line of the Main screen. (The following words are pushed to the next line, where they are displayed as a paragraph with word wrap.) 
5. For experiments of this kind, the Nav window should be turned off by default. Probably the Tool panel and Help panel should also be turned off, and the Menu too.

**Comment by kayaulai** [comment]
If we are going to make this interface, perhaps it can be applied to data that is imported and organised in terms of turns, but with no unit information? After all, I'm sure there will be a lot of conversational transcripts that don't do IUs, and since we're providing an interface for separating turns into IUs, why not allow annotators to use it as well, and not just participants in our (somewhat particular) experiment?

**Comment by johnwdubois** [comment]
Yes, this should be a tool available to everyone, not just for experiments. If the user has a transcription that shows turns but no IUs, they could use Rezonator to mark the IUs. It could also be used for other units. it could also be used as a tutorial, with a hidden gold standard segmentation into IUs. The user does their best, then gets feedback.
